app/views/layouts/application.html.erb

```diff
@@ -90,6 +87,16 @@
               <%= render 'shared/header_navbar' %>
             </div>
           </div>
+          <div class="navbar collapse navbar-collapse" aria-label="browse" id="user-util-collapse">
```

Comment:
Do we need the `aria-label` here?

Reply:
No. Just following the component library

Reply:
In the component library `.navbar-collapse` is on a `<nav>` element, which requires an `aria-label`